share/smack/frontend.py

```diff
@@ -249,16 +251,17 @@ def json_compilation_database_frontend(input_file, args):
                 bit_codes = [re.sub('[.]o$', '.bc', f) for f in cc['objects']]
                 try_command(['llvm-link', '-o', args.bc_file] + bit_codes)
                 try_command(['llvm-link', '-o', args.linked_bc_file,
-                             args.bc_file] + build_libs(args))
+                             args.bc_file] + default_build_libs(args))
 
             else:
-                out_file = output_flags.findall(cc['command'])[0] + '.bc'
+                # out_file = output_flags.findall(cc['command'])[0] + '.bc'
```

Comment:
Why did you comment out this one?

Reply:
`out_file` was never used and it led to a warning.

Reply:
So why not delete it completely?

Reply:
> So why not delete it completely?

I thought maybe this line contains some information may be useful to revive this feature.

Reply:
What does it do? What is its purpose?

Reply:
Ok, we agreed to remove this line.